### PR TITLE
[REVIEW] Aligning signature of `Series.value_counts` to Pandas

### LINKED ISSUE #5417: [FEA] Align signature of `Series.value_counts` to Pandas
Currently we only support `sort` flag, but there's an `ascending` kwargthat we should be able to support easily as well as the `bins` and `dropna` kwarg which we can throw if a user sets an unsupported value on.

### PR DESCRIPTION
**Pull request by rgsl888prabhu**
closes #5417

### CODE REVIEW

File: python/cudf/cudf/core/series.py [hunk 1]
```diff
@@ -3349,18 +3349,71 @@ def nunique(self, method="sort", dropna=True):
...
+        >>> sr = cudf.Series([1.0, 2.0, 2.0, 3.0, 3.0, 3.0, None])
+        >>> sr.value_counts(ascending=True)
+        1.0    1
+        2.0    2
+        3.0    3
+        dtype: int32
         """
 
+        if normalize is True:
+            raise NotImplementedError("normalize == True is not supported")
```
Comment (kkraus14) [inline python/cudf/cudf/core/series.py:3403]:
```suggestion
        if normalize != False:
            raise NotImplementedError("Only normalize == False is currently supported")
```

File: python/cudf/cudf/core/series.py [hunk 2]
```diff
@@ -3349,18 +3349,71 @@ def nunique(self, method="sort", dropna=True):
...
+        if dropna is False:
+            raise NotImplementedError("dropna == False is not supported")
```
Comment (kkraus14) [inline python/cudf/cudf/core/series.py:3405]:
```suggestion
        if dropna != True:
            raise NotImplementedError("Only dropna ==True is currently supported")
```

File: python/cudf/cudf/core/series.py [hunk 3]
```diff
@@ -3349,18 +3349,71 @@ def nunique(self, method="sort", dropna=True):
...
+        if bins is not None:
+            raise NotImplementedError("bins is not supported")
```
Comment (kkraus14) [inline python/cudf/cudf/core/series.py:3407]:
```suggestion
            raise NotImplementedError("bins is not yet supported")
```

File: python/cudf/cudf/core/series.py [hunk 4]
```diff
@@ -3399,10 +3399,14 @@ def value_counts(
...
-            raise NotImplementedError("Only normalize == False is currently supported")
-        if dropna != True:
-            raise NotImplementedError("Only dropna ==True is currently supported")
+        if normalize is not False:
+            raise NotImplementedError(
+                "Only normalize == False is currently supported"
+            )
+        if dropna is not True:
+            raise NotImplementedError(
+                "Only dropna ==True is currently supported"
```
Comment (kkraus14) [inline python/cudf/cudf/core/series.py:3408]:
```suggestion
                "Only dropna == True is currently supported"
```

Comment (kkraus14) [reply python/cudf/cudf/core/series.py:3408]: We can just apply this after CI passes and I'll admin merge.

Comment (rgsl888prabhu) [reply python/cudf/cudf/core/series.py]: I was too fast and impatient :)

**Pull request merged**